### ISSUE TITLE
Domain authentication steps for GODADDY are incorrect

### ISSUE THREAD

**Issue by jonesjsc** [sendgrid/docs#6530]
Sender Authentication -> Domain Authentication

Instructions specific to GoDaddy DNS hosting - the instructions say that the host value should include the root domain.

For example - example.com

The instructions for godaddy.com say that the 3 CNAME HOST / Values should be
| TYPE | HOST | VALUE |
|-------|:-------|:--------|
|CNAME|em####.example.com|u11111111.wl222.sendgrid.net|
|CNAME|s1._domainkey.example.com|s1.domainkey.u11111111.wl222.sendgrid.net|
|CNAME|s2._domainkey.example.com |s2.domainkey.u22222222.wl222.sendgrid.net|

For godaddy, this is incorrect.

These instructions for godaddy should instead be:

| TYPE | HOST | VALUE |
|-------|:-------|:--------|
|CNAME|em####|u11111111.wl222.sendgrid.net|
|CNAME|s1._domainkey|s1.domainkey.u11111111.wl222.sendgrid.net|
|CNAME|s2._domainkey|s2.domainkey.u22222222.wl222.sendgrid.net|

## Supporting link

https://www.godaddy.com/community/Managing-Domains/DNS-TXT-CNAME-records-are-still-not-getting-propagate/m-p/153099

**Comment by gabbello** [comment]
Yes, this should be made much more clear in sendgrid docs, I lost 2 days trying to configure it. There is the notice that some providers do this, but I would have expected that f they do this it is also visually visible so it would show smth like: s1._domainkey.example.com.example.com, but this is not the case with godaddy. So make it clear in your docs that godaddy does this.
